data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3482 @@
+			`	Considering all of this, the Unfettered may not take no for an answer... Plenty of ships are already taking off to try and get those much needed medicals.`
+			choice
+				`	(Signal them I am going to get the supplies.)`
+					accept
+				`	(I do not have time for this.)`
+			`	Thankfully it seems that everyone is too busy with the local problem to annoy you or press you into accepting this mission, but this will probably have consequences later. You better not stay landed for too long, or you could draw unwanted attention.`
+	
+	on visit
+		dialog phrase "generic cargo on visit"
+	on stopover
+		dialog `You are surprised to see the Hai are more than happy to give you the medical supplies needed. They load up the cargo and once they are done one of they addresses you, "Tell our brothers and sisters to stay strong, hopefully it is not too late," he says with determination.`
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	
+	on complete
+		log `Brought medial supplies to Darkcloak when it was in great need of them, to prevent an epidemic.`
+		log "People" "Sha-Ri" `An Unfettered Chief worried for his people, inside Hai or Unfettered territory. He seems to believe that all Hai are equal, and as a member of the council he probably has quite some influence.`
+		payment 1500000
+		conversation
+			`The local health specialists seem greatly relieved when you manage to bring the medical supplies back in time. They signal you quickly to the local admninistrators who give you <payment>.`
+			`	As you head to leave, you notice the same Hai Chief speaking with the medical teams. He soon sees you as well and moves to speak with you, seeming more than a bit emotional, "I knew I could count on you! With the swift response of captains like you, it seems like it has calmed down, maybe we worried too much. We did not before have the time to get properly introduced. I am Sha-Ri." He makes a little gesture, you assume of deference. You try to imitate the motion, apparently badly, amusing Sha-Ri. "Worry not, <first> <last>, I already know who you are."`
+			choice
+				`	"I only did what little I could do to help save people. Hopefully I will be able to do more next time we meet."`
+					accept
+				`	"Why was the message only addressed to Hai?"`
+				`	"Were these supplies for soldiers?"`
+					goto soldiers
+			`	He does not seem that much surprised, only disappointed, "I had no say in this, only idiots would refuse help when we are in such need. They will probably say aliens are the reason of this epidemic in the first place... this is not the kind of thinking that will make us strong again, I can tell you that much."`
```

Comment:
This whole option is actually pretty nonsensical.

Reply:
You mean that they would not accept help from an alien? I mean they shoot you on sight, and from their perspective aliens are the ones who gave them the disease, I don't see how it doesn't make sense they would be even more distant with aliens; at least for some and for a bit of time.